### PR TITLE
Homepage, template and navigation

### PR DESCRIPTION
**Pull request by deadlight**
## Done
Basic Vanilla configuration, base template and homepage

## QA
 - ./run
 - Visit http://0.0.0.0:8031

### CODE REVIEW

**Comment by anasereijo** [comment]
see below some changes. should be quite straightforward (sorry if not!)

landscape home

**Comment by anasereijo** [comment]
@deadlight awesome! thanks 👍

**Comment by anthonydillon** [comment]
@deadlight what is going on with this branch? Is it in review?

File: static/sass/_patterns_navigation.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@mixin landscape-navigation {
+  .p-navigation__link > a:hover {
+    background-color: #5E2750;
```
Comment (anthonydillon) [inline static/sass/_patterns_navigation.scss:6]: I would suggest replacing with:
```scss
@mixin vf-navigation-default {
  .p-navigation {
    @include vf-navigation-pattern(
      $background-color: $color-navigation-background,
      $border-color: if(lightness($color-navigation-background) > 50, $color-mid-light, transparent),
      $text-color: #5E2750
    );
  }
}
```

File: templates/_layout.html [hunk 2]
```diff
@@ -102,28 +79,75 @@
     </div>
     </div>
   </div>
-</script>
+</script> -->
```
Comment (anthonydillon) [inline templates/_layout.html]: Please remove commented out code.

**Comment by deadlight** [comment]
@anthonydillon it is now

File: templates/_layout.html [hunk 3]
```diff
@@ -102,28 +79,75 @@
...
+              <a href="https://help.landscape.canonical.com/" class="footer__link">
+                Help</a>
+            </li>
+          </ul>
+        </div>
+        <div class="col-5">
+          <h3 class="p-muted-heading">Need help?</h3>
+          <ul class="p-list">
+            <li class="p-list__item">
+              <a href="https://www.ubuntu.com/management/ubuntu-advantage" class="external">Learn more about Ubuntu Advantage</a>
```
Comment (anthonydillon) [inline templates/_layout.html]: https://www.ubuntu.com/support/plans-and-pricing

File: templates/_layout.html [hunk 4]
```diff
@@ -102,28 +79,75 @@
...
+          </ul>
+        </div>
+        <div class="col-4">
+          <h3 class="p-muted-heading">Already a customer?</h3>
+          <ul class="p-list">
+            <li class="p-list__item">
+              <a href="login/authenticate">Log in&nbsp;&rsaquo;</a>
+            </li>
+          </ul>
+          <p>If you have any problems accessing your account, please <a href="https://www.ubuntu.com/about/contact-us">contact us</a>.
```
Comment (anthonydillon) [inline templates/_layout.html]: https://www.ubuntu.com/contact-us

File: templates/landscape/index.html [hunk 5]
```diff
@@ -1,8 +1,150 @@
...
+    <ul class="p-list is-split">
+      <li class="p-list__item is-ticked">Automate day-to-day tasks</li>
+      <li class="p-list__item is-ticked">Receive alerts to update machines you specify</li>
+      <li class="p-list__item is-ticked">Keep secure with the latest security patches</li>
+      <li class="p-list__item is-ticked">Manage up to 40,000 machines with a single interface</li>
+      <li class="p-list__item is-ticked">Create your own software repositories</li>
+      <li class="p-list__item is-ticked">Extend and customise Landscape via our API</li>
+    </ul>
+    <div class="u-sv2">
+      <img class="" src="https://assets.ubuntu.com/v1/44ac1c31-web-ui-2.gif" alt="Landscape admin splashscreen">
```
Comment (anthonydillon) [inline templates/landscape/index.html]: empty class

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some comments inline

**Comment by anthonydillon** [comment]
QA issue:
- Can you remove the `max-height` and `background-image` from small screens
screenshot_2018-11-08 landscape

**Comment by deadlight** [comment]
@anthonydillon ready for re-review

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**